packages/yourloops/pages/team/team-details-page.tsx

```diff
@@ -122,6 +129,10 @@ function TeamDetailsPage(): JSX.Element {
     refresh()
   }, [refresh])
 
+  React.useEffect(() => {
+    setPageTitle(t('header-tab-care-team'))
+  }, [t])
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** Hmmm I am suprised by this. Can `t` dependency actually change? Do we need this code?‏